src/GraphQL.SystemTextJson/GraphQL.SystemTextJson.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net6</TargetFrameworks>
```

Comment:
New API capabilities?

Reply:
Yes, for optimized `ReadNode<T>(...)` behavior.